build-configs/integreat-e2e/index.ts

```diff
@@ -1,4 +1,4 @@
-import integreatTestCmsPlatformBuildConfigs from '../integreat-test-cms'
+import integreatPlatformBuildConfigs from '../integreat'
```

Comment:
Why do we now use integreat as base?

Reply:
The testumgebung-e2e instance is only on the live cms.

Reply:
There are other build config settings that differ between integreat and integreat-test-cms, did you check whether that are all okay?

Reply:
for web I don't think there will be any problem. But can you check the app settings? Not sure whether the google stuff messes with anything.